content/java/concepts/strings/terms/getChars.md

```diff
@@ -0,0 +1,59 @@
+---
+Title: '.getChars()'
+Description: ' A java strings method that copies characters from the given string into the destination character array.'
+Subjects:
+  - 'Computer Science'
+  - 'Code Foundations'
+Tags:
+  - 'Strings'
+  - 'Methods'
+CatalogContent:
+  - 'learn-c-sharp'
+  - 'paths/computer-science'
+---
+
+The **.getChars()** string method copies a set of characters from an array. The method then copies the characters into character array.
+
+## **Synthax**
+
+```pseudo
+str.getChars(12, 25, destArray, 0);
+```
+
+'str' is the string that is being converted to a new array 
+
+## **Example
+
+In the following example, 'getChars()' converts a 'string' into a new array. 
+
+```cs
```

Comment:
Several things here.  First off you misplaced the end of the code block, Second, you need to test your code to make sure the example works. Third, you are writing a Java string method, not the c# method. Your syntax and examples need to be in Java.